load/data_loader.py

```diff
@@ -1,4 +1,5 @@
 import torch.multiprocessing as mp
+from sampling.sample_creator import SampleCreator
```

Comment:
The reason we called the file `load.py` is that the user API will be something like one_access.load.DataLoader(...). Now it will be one_access.data_loader.DataLoader(...). The first sounds better to me, what do you think?

Reply:
I agree. Will change back